modules/creating-a-machine-pool-cli.adoc

```diff
@@ -36,6 +39,9 @@ $ rosa create machinepool --cluster=<cluster-name> \
 <5> Optional: Defines the taints for the machine pool. Replace `<key>=<value>:<effect>,<key>=<value>:<effect>` with a key, value, and effect for each taint, for example `--taints=key1=value1:NoSchedule,key2=value2:NoExecute`. Available effects include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`.
 <6> Optional: Configures your machine pool to deploy machines as non-guaranteed AWS Spot Instances. For information, see link:https://aws.amazon.com/ec2/spot/[Amazon EC2 Spot Instances] in the AWS documentation. If you select *Use Amazon EC2 Spot Instances* for a machine pool, you cannot disable the option after the machine pool is created.
 <7> Optional: If you have opted to use Spot Instances, you can specify this argument to define a maximum hourly price for a Spot Instance. If this argument is not specified, the on-demand price is used.
+ifdef::openshift-rosa[]
+<8> Optional: Specifies the worker node disk size.
```

Comment:
Add more introduction with an example value with unit sould be better like ` Root disk size with a suffix like GiB or TiB`